src/Mvc/Mvc.Core/src/ProducesResponseTypeAttribute.cs

```diff
@@ -43,6 +45,8 @@ public ProducesResponseTypeAttribute(Type type, int statusCode)
         /// </summary>
         public int StatusCode { get; set; }
 
+        internal bool SetByDefault { get; private set; }
```

Comment:
You don't need to document `internal` apis, but it never hurts to add one. Also, `IsResponseTypeImplicitlyAssigned` or `IsResponseTypeSetByDefault`

Reply:
```suggestion
        internal bool SetByDefault { get; }
```